src/device.rs

```diff
@@ -218,12 +221,17 @@ where
                             return Err(RequestError::ClientError(err));
                         }
                         Ok(res) => {
+                            // This return type is defined in https://tools.ietf.org/html/draft-ietf-oauth-device-flow-07#section-3.2
+                            // The alias is present as Google use a non-standard name for verification_uri.
```

Comment:
nasty!

Reply:
Yup! Welcome to software!

I've updated this link as that's an old version of the spec. Of course, it's still draft too!